docs/chart_template_guide/variables.md

```diff
@@ -114,6 +114,8 @@ metadata:
     # I cannot reference .Chart.Name, but I can do $.Chart.Name
     helm.sh/chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
     app.kubernetes.io/instance: "{{ $.Release.Name }}"
+    # Value from appVersion in Chart.yaml
+    app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
```

Comment:
This is a nit but the other docs in this section start with `$.Chart...`. This instance should hold the pattern to be consistent. Can you please update it.

Reply:
updated.